project.clj

```diff
@@ -1,4 +1,4 @@
-(defproject tech.gojek/bulwark "1.1.0"
+(defproject tech.gojek/bulwark "1.1.1"
```

Comment:
Let's call it `1.2.0`. Please update the changelog as well -https://github.com/gojek/bulwark/blob/master/CHANGELOG.md